django/src/games/pong.py

```diff
@@ -214,8 +225,9 @@ async def save_pong_to_db(self, winner):
                 score1=self.info.score[0],
                 score2=self.info.score[1]
             )
+            logger.debug(f"Game {self.room_id} saved to db.")
         except Exception as e:
-            print(f"Error when saving game {self.room_id} to db: {e}")
+            logger.error(f"Cannot save game {self.room_id} to db: {e}")
```

Comment:
All error messages above use past tense, stay logical
use "Could not save game _roomid_ to db: ...."